### ISSUE TITLE
(FED) Keyboard focus state not visible when user navigates to location in PSW

### ISSUE THREAD

**Issue by MichaelGrahamVS** [visitscotland/vs-component-library#237]
### Raised by
GDS

### Describe the bug
On the homepage, the keyboard focus is not visible when the keyboard tabs through the dropdown menu under 'Location'.

### To Reproduce
1. Go to (https://www.visitscotland.com/)
2. Tab to PSW
3. Navigate with the keyboard to Location and hit down
4. Keyboard focus is not visible

### Expected behavior
Keyboard focus state should be visible for the user

### Page URL
[URL of page where the bug exists](https://www.visitscotland.com/)

### Environment
Live

### Browser
All

### Device and version
All

### Impact on end user
Difficult navigate using keyboard

### Additional context
https://github.com/visitscotland/dot-com-brxm/issues/238
